Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6-slim-buster
+FROM python:3.6-slim-stretch
```

Comment:
`openjdk-8-jdk` is no more available in Debian buster.

Reply:
![](https://media.giphy.com/media/4oy28BX8lfGKrF3rnU/giphy.gif)